website/src/pages/mapper/dev_guide/deployment/azure/steps.mdx

```diff
@@ -0,0 +1,421 @@
+
+<Tabs items={['Frontend app', 'Backend app']}>
+  <Tabs.Tab>
+  Access type: `Public`
+
+Registry server URL:
+
+```bash
+ghcr.io/health-informatics-uon
+```
+
+Image and tag:
+
+```bash
+carrot/frontend:<tag>
+```
+
+You can choose a version of this container by using the tag for the image. Look for available tags [here](https://github.com/Health-Informatics-UoN/carrot-mapper/pkgs/container/carrot%2Ffrontend).
+
+  </Tabs.Tab>
+  <Tabs.Tab>
+
+Access type: `Public`
+
+Registry server URL:
+
+```bash
+ghcr.io/health-informatics-uon
+```
+
+Image and tag:
+
+```bash
+carrot/backend:<tag>
+```
+
+You can choose a version of this container by using the tag for the image. Look for available tags [here](https://github.com/Health-Informatics-UoN/carrot-mapper/pkgs/container/carrot%2Fbackend).
+
+  </Tabs.Tab>
+</Tabs>
+
+**Networking**: As default.
+
+**Monitor + Secure**: We recommend Application Insights to be enabled.
+
+**Tags**: Give name and value for the web app's tag, if needed.
+
+### Function app
+
+Choose `App service` when prompted.
+
+**Basics**
+
+Choose a suitable `Subscription`, `Resource group`, `Region`, `Pricing plan`, and a name for your Function app.
+
+Select `Container Image` as the mode of deploying.
+
+**Storage**: Choose or create an Azure storage where the Function app will store its Blobs and Queues.
+
+**Networking**: As default.
+
+**Monitor + Secure**: We recommend Application Insights to be enabled.
+
+**Tags**: Give name and value for the web app's tag, if needed.
+
+**Container Image**: A default image will be used as you create a function app. Therefore, after the app being deployed you must change this image by doing so:
+
+- Access the newly created Function app
+- Under tab `Deployment`, choose `Deployment center`
+- Choose `Private Registry` as Registry source
+- Change Server URL to: `https://ghcr.io`
+- Change Full Image Name and Tag to: `health-informatics-uon/carrot/workers:<tag>`. Look for available tags [here](https://github.com/Health-Informatics-UoN/carrot-mapper/pkgs/container/carrot%2Fworkers).
+- Save the setting
+
+### PostgreSQL Database
+
+If you didn't have one, create a PostgreSQL Database on Azure.
+
+Follow prompted steps to create a suitable Database for Carrot.
+
+### Data Factory
+
+Data factory is useful when creating workflows/scripts to load and manage data on Azure storage and Azure database.
+
+Create one following the prompted steps.
+
+### Key vault
+
+Key vault is highly recommended to keep secrets and keys for functioning Carrot safe. If you didn't have one, create a Key Vault following the prompted steps on Azure.
+
+</Steps>
+
+## Preparation
+
+You will need to prepare your database and storage before configure other apps.
+
+<Steps>
+
+### Database
+
+Using Data factory or other ways to create a schema named `omop` in your database, then load there the vocabularies, which you would like in Carrot, downloaded from [Athena](https://athena.ohdsi.org/vocabulary/list).
```

Comment:
Could we link out to OHDSI for their guidance on this https://github.com/OHDSI/Vocabulary-v5.0/wiki/General-Structure,-Download-and-Use